gammapy/data/data_store.py

```diff
@@ -71,12 +71,14 @@ def from_dir(cls, base_dir, hdu_table_filename=None, obs_table_filename=None):
         base_dir = make_path(base_dir)
 
         if hdu_table_filename:
+            hdu_table_filename = make_path(hdu_table_filename)
             if Path.exists(base_dir / hdu_table_filename):
```

Comment:
You are still calling `Path.exists` as a static function.
That method should only be called on Path object instances.
Maybe like this here:
```
(base_dir / hdu_table_filename).exists()
```